### DISCUSSION TITLE
Installation Error

### DISCUSSION THREAD

**Discussion by Devmatcot** [appwrite/appwrite#3786] [Q&A] [answered]
please can I get help to fix this error?
I have tried severally to install appwrite locally with docker this is the error all the time

Warning: yaml_parse(): reading error encountered during parsing: control characters are not allowed (line 1, column 1) in /usr/src/code/src/Appwrite/Docker/Compose.php on line 20
Service not found

**Comment by stnguyen90** [comment, marked as answer]
@Devmatcot, the error indicates there is something wrong with your docker-compose.yml file. How did you generate it? How did you install Appwrite?

**Reply by Devmatcot** [reply]
i installed it with provide appwrite installation command for power shell

**Reply by stnguyen90** [reply]
@Devmatcot did you modify it at all after? Can you share the file?

**Reply by Devmatcot** [reply]
No, I don't modify anything, I don't even know where the file is on my PC, after I installed docker desktop, then I copy the appwrite installation code into my terminal and I docker get the images but I always got that error

I appreciate your response to my request

**Reply by stnguyen90** [reply]
@Devmatcot look to where you executed the code and see if there is an appwrite folder. If not, that could be the problem. You may be executing it in a place you don't have write permissions. Try changing to some other directory like your Documents folder.

**Reply by stnguyen90** [reply]
@Devmatcot are you still having issues?

**Reply by Devmatcot** [reply]
i try it on my friend's PC and it was successful I found out that I don't have write permission on my hard drive.
thanks for your response
